.travis.yml

```diff
@@ -1,16 +1,17 @@
 language: go
 go:
-  - "1.10"
+  - "1.11"
   - "tip"
 
 before_install:
   - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
 install:
-  - dep ensure
+  - GO111MODULE="on" go mod tidy
```

Comment:
👋 hey, we should not run` got mod tidy` in the CI, I have this template repo that you can get some inspiration on how to run on Travis Go modules projects. Hope it's helpful 😄   https://github.com/geototti21/go-travis

Reply:
Thank you, I am aware of that, but was trying to solve some dependencies issues that this created with go11, it didn't work. If you want to take a look at the travis error and give an advice on how to resolve it it will be helpful to proceed with this PR.

Reply:
@geototti21 
```
go build github.com/aws/aws-lambda-go/events: module requires Go 1.12
go build github.com/onsi/ginkgo/config: module requires Go 1.13
go build github.com/onsi/ginkgo/types: module requires Go 1.13
# google.golang.org/protobuf/proto
../../../../pkg/mod/google.golang.org/protobuf@v1.23.0/proto/proto_methods.go:18:23: cannot use m.ProtoMethods() (type *protoreflect.Message) as type *struct { pragma.NoUnkeyedLiterals; Flags uint64; Size func(struct { pragma.NoUnkeyedLiterals; Message protoreflect.Message; Flags uint8 }) struct { pragma.NoUnkeyedLiterals; Size int }; Marshal func(struct { pragma.NoUnkeyedLiterals; Message protoreflect.Message; Buf []byte; Flags uint8 }) (struct { pragma.NoUnkeyedLiterals; Buf []byte }, error); Unmarshal func(struct { pragma.NoUnkeyedLiterals; Message protoreflect.Message; Buf []byte; Flags uint8; Resolver interface { FindExtensionByName(protoreflect.FullName) (protoreflect.ExtensionType, error); FindExtensionByNumber(protoreflect.FullName, protowire.Number) (protoreflect.ExtensionType, error) } }) (struct { pragma.NoUnkeyedLiterals; Flags uint8 }, error); Merge func(struct { pragma.NoUnkeyedLiterals; Source protoreflect.Message; Destination protoreflect.Message }) struct { pragma.NoUnkeyedLiterals; Flags uint8 }; CheckInitialized func(struct { pragma.NoUnkeyedLiterals; Message protoreflect.Message }) (struct { pragma.NoUnkeyedLiterals }, error) } in return argument
go build github.com/onsi/ginkgo/internal/writer: module requires Go 1.13
go build github.com/nxadm/tail/ratelimiter: module requires Go 1.13
go build github.com/nxadm/tail/util: module requires Go 1.13
go build golang.org/x/sys/internal/unsafeheader: module requires Go 1.12
go build github.com/onsi/ginkgo/internal/testingtproxy: module requires Go 1.13
go build github.com/onsi/ginkgo/reporters/stenographer/support/go-colorable: module requires Go 1.13
go build github.com/aws/aws-lambda-go/lambdacontext: module requires Go 1.12
go build github.com/jmespath/go-jmespath: module requires Go 1.14
go build github.com/aws/aws-lambda-go/lambda/handlertrace: module requires Go 1.12
go build github.com/aws/aws-lambda-go/lambda/messages: module requires Go 1.12
FAIL	github.com/epsagon/epsagon-go/epsagon [build failed]
FAIL	github.com/epsagon/epsagon-go/wrappers/aws/aws-sdk-go/aws [build failed]
FAIL	github.com/epsagon/epsagon-go/wrappers/net/http [build failed]
```
Any idea why this is happening? (The protobuf one is probably unrelated version issue.)

Reply:
yeah I think so, can you please use `"1.11.x"` on the travis. I think there were some issues that have been patched and requires the latest minor version of 1.11

Reply:
@geototti21 Thanks, this worked.